graphql/schema.graphql

```diff
@@ -498,6 +504,31 @@ input TaskSyncOptionsInput {
   patchEnabled: Boolean!
 }
 
+input BuildBaronSettingsInput {
+  ticketCreateProject: String!
+  ticketSearchProjects: [String!]
+  bfSuggestionServer: String!
+  bfSuggestionUsername: String!
+  bfSuggestionPassword: String!
+  bfSuggestionTimeoutSecs: Int!
+  bfSuggestionFeaturesURL: String!
```

Comment:
Just to validate that you are aware of this and that you are intentionally adding the `!`. the `!` means that this field is required. And will not pass validation unless those fields are included in the request.  So a user would not be able to partially update one of these fields and would be required to pass in the full object when ever passing this input